Test cases for `processors/ayon_hiero_effect_file.py`

jakubjezek001 opened the pull request:
Adding more coverage for `ayon_hiero_effect_file.py` file. 

## TODO:
- [x] Add testing dummy files

jakubjezek001 committed:
adding more test cases

tweak-wtf commented on tests/test_processors_ayon_hiero_effect_file.py:
```diff
@@ -120,3 +121,57 @@ def test_EffectsFileProcessor(self, kwargs, results):
+
+    def test_clear_operators(self):
+        effect_processor = AYONHieroEffectsFileProcessor(
+            Path("dummy.json")
```
seems this `dummy.json` isn't available in the repo.
https://github.com/ynput/LabLib/actions/runs/12292170415/job/34302276633#step:9:272

let's push this into `resources/public`

jakubjezek001 replied:
ah, yeah. It seems like it is actually missing even on my machine. Will look into the code again and see if it is actually needed anymore.

tweak-wtf replied:
alrighty. thanks for checking this out 🙏

jakubjezek001 committed:
Add test command to help message

- Included a new option for running tests
- Updated help output to reflect the change

jakubjezek001 committed:
Update tests for effect file processor

- Refactored test to use temporary files
- Improved handling of invalid JSON and missing files
- Adjusted assertions for operator clearing and loading
- Enhanced operation sorting with additional track index info

tweak-wtf reviewed (commented):
perfecto :)
